modules/network/application-gateway/.test/common/main.test.bicep

```diff
@@ -39,7 +42,7 @@ module nestedDependencies 'dependencies.bicep' = {
     publicIPName: 'dep-${namePrefix}-pip-${serviceShort}'
     managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
     certDeploymentScriptName: 'dep-${namePrefix}-ds-${serviceShort}'
-    keyVaultName: 'dep-${namePrefix}-kv-${serviceShort}'
+    keyVaultName: 'dep-${namePrefix}-kv-${serviceShort}-${substring(uniqueString(baseTime), 0, 3)}'
```

Comment:
Why the unique name? The far we only needed it in cases where purge protection was required

Reply:
I ran into an error with the key vault being in a soft delete state, but must have been due to a manual deletion. I see that the removal pipeline process handles that. I'll go ahead and revert that change for the key vault name.